src/ralph/admin/sites.py

```diff
@@ -63,11 +65,32 @@ def get_urls(self, *args, **kwargs):
                         *get_urls_chunks(model, view)
                     ),
                     view.as_view(),
-                    {'model': model},
+                    {'model': model, 'views': model_admin.change_views},
                     name='{}_{}_{}'.format(*get_urls_chunks(model, view))
                 ))
         return urlpatterns
 
+    def get_extra_view_menu_items(self):
+        """Method returns list of items for sitetree mechanism."""
+        items = defaultdict(list)
+
+        def get_item(model, view, change_view=False):
+            url = 'admin:{}_{}_{}'.format(
```

Comment:
This is next time I see such code, maybe we could add a helper for that?
Simple function is enough. What do you think?